src/components/DisplayTaxonName.js

```diff
@@ -34,21 +34,13 @@ const uncapitalized = new Set( [
   // On the web, we use toUpperCase to generate this at runtime, but that
   // seems to be causing a bug in the Android JS engine as of 20230110, so
   // we're hard-coding it here. ~~~kueda 20220110
   // eslint-disable-next-line max-len
-  const upperCaseChars = "ΜSSÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØØÙÚÛÜÝÞŸĀĂĄĆĈĊČĎĒĔĖĘĚĜĞĠĢĤĨĪĬĮĴĶĹĻĽŁŃŅŇŌŎŐŒŒŔŖŘŚŜŞŠŢŤŨŪŬŮŰŲŴŶŹŻŽƠƯǍǏǑǓǕǗǙǛǞǠǢǦǨǪǬǮJ̌ǴǸǺǼǾȀȂȄȆȈȊȌȎȐȒȔȖȘȚȞȦȨȪȬȮȰȲΩḀḂḄḆḈḊḌḎḐḒḔḖḘḚḜḞḠḢḤḦḨḪḬḮḰḲḴḶḸḺḼḾṀṂṄṆṈṊṌṎṐṒṔṖṘṚṜṞṠṢṤṦṨṪṬṮṰṲṴṶṸṺṼṾẀẂẄẆẈẊẌẎẐẒẔH̱T̈W̊Y̊ṠẠẢẤẦẨẪẬẮẰẲẴẶẸẺẼẾỀỂỄỆỈỊỌỎỐỒỔỖỘỚỜỞỠỢỤỦỨỪỬỮỰỲỴỶỸ∂∆∑FIFL";
+  const upperCaseChars = "\u{39C}SS\u{C0}\u{C1}\u{C2}\u{C3}\u{C4}\u{C5}\u{C6}\u{C7}\u{C8}\u{C9}\u{CA}\u{CB}\u{CC}\u{CD}\u{CE}\u{CF}\u{D0}\u{D1}\u{D2}\u{D3}\u{D4}\u{D5}\u{D6}\u{D8}\u{D8}\u{D9}\u{DA}\u{DB}\u{DC}\u{DD}\u{DE}\u{178}\u{100}\u{102}\u{104}\u{106}\u{108}\u{10A}\u{10C}\u{10E}\u{112}\u{114}\u{116}\u{118}\u{11A}\u{11C}\u{11E}\u{120}\u{122}\u{124}\u{128}\u{12A}\u{12C}\u{12E}\u{134}\u{136}\u{139}\u{13B}\u{13D}\u{141}\u{143}\u{145}\u{147}\u{14C}\u{14E}\u{150}\u{152}\u{152}\u{154}\u{156}\u{158}\u{15A}\u{15C}\u{15E}\u{160}\u{162}\u{164}\u{168}\u{16A}\u{16C}\u{16E}\u{170}\u{172}\u{174}\u{176}\u{179}\u{17B}\u{17D}\u{1A0}\u{1AF}\u{1CD}\u{1CF}\u{1D1}\u{1D3}\u{1D5}\u{1D7}\u{1D9}\u{1DB}\u{1DE}\u{1E0}\u{1E2}\u{1E6}\u{1E8}\u{1EA}\u{1EC}\u{1EE}J\u{30C}\u{1F4}\u{1F8}\u{1FA}\u{1FC}\u{1FE}\u{200}\u{202}\u{204}\u{206}\u{208}\u{20A}\u{20C}\u{20E}\u{210}\u{212}\u{214}\u{216}\u{218}\u{21A}\u{21E}\u{226}\u{228}\u{22A}\u{22C}\u{22E}\u{230}\u{232}\u{3A9}\u{1E00}\u{1E02}\u{1E04}\u{1E06}\u{1E08}\u{1E0A}\u{1E0C}\u{1E0E}\u{1E10}\u{1E12}\u{1E14}\u{1E16}\u{1E18}\u{1E1A}\u{1E1C}\u{1E1E}\u{1E20}\u{1E22}\u{1E24}\u{1E26}\u{1E28}\u{1E2A}\u{1E2C}\u{1E2E}\u{1E30}\u{1E32}\u{1E34}\u{1E36}\u{1E38}\u{1E3A}\u{1E3C}\u{1E3E}\u{1E40}\u{1E42}\u{1E44}\u{1E46}\u{1E48}\u{1E4A}\u{1E4C}\u{1E4E}\u{1E50}\u{1E52}\u{1E54}\u{1E56}\u{1E58}\u{1E5A}\u{1E5C}\u{1E5E}\u{1E60}\u{1E62}\u{1E64}\u{1E66}\u{1E68}\u{1E6A}\u{1E6C}\u{1E6E}\u{1E70}\u{1E72}\u{1E74}\u{1E76}\u{1E78}\u{1E7A}\u{1E7C}\u{1E7E}\u{1E80}\u{1E82}\u{1E84}\u{1E86}\u{1E88}\u{1E8A}\u{1E8C}\u{1E8E}\u{1E90}\u{1E92}\u{1E94}H\u{331}T\u{308}W\u{30A}Y\u{30A}\u{1E60}\u{1EA0}\u{1EA2}\u{1EA4}\u{1EA6}\u{1EA8}\u{1EAA}\u{1EAC}\u{1EAE}\u{1EB0}\u{1EB2}\u{1EB4}\u{1EB6}\u{1EB8}\u{1EBA}\u{1EBC}\u{1EBE}\u{1EC0}\u{1EC2}\u{1EC4}\u{1EC6}\u{1EC8}\u{1ECA}\u{1ECC}\u{1ECE}\u{1ED0}\u{1ED2}\u{1ED4}\u{1ED6}\u{1ED8}\u{1EDA}\u{1EDC}\u{1EDE}\u{1EE0}\u{1EE2}\u{1EE4}\u{1EE6}\u{1EE8}\u{1EEA}\u{1EEC}\u{1EEE}\u{1EF0}\u{1EF2}\u{1EF4}\u{1EF6}\u{1EF8}\u{2202}\u{2206}\u{2211}FIFL";
```

Comment:
I haven't tested this yet, but do we need to remove the "SS" an "FIFL" from this string? Or otherwise something that starts with any of those characters will not be replaced?

Reply:
I'm unsure since those were in the original charset as well